package.json

```diff
@@ -62,6 +63,8 @@
     "aurelia-route-recognizer": "^1.2.0"
   },
   "devDependencies": {
+    "@types/jasmine": "^2.8.8",
+    "aurelia-templating": "^1.8.2",
```

Comment:
What's the need for adding the templating library here?

Reply:
@EisenbergEffect It's for `import { CompositionContext } from 'aurelia-templating'` for better intellisense.